popup/popup.html

```diff
@@ -61,6 +61,21 @@
                     <p></p>
                 </div>
             </div>
+            <div class="section">
+                <p class="section-heading" title="jira.tools.sap">SAP Jira</p>
+                <div class="section-text disabled"
+                    data-dynamic-content-script-name="jira.tools.sap"
+                    title="Show the count tips on the right side of the pagination.">
+                    <p>Show Count Tips</p>
+                    <p>
+                        <label class="switch" title="Optional: jira.tools.sap (You may enable this first within 'Optional Permissions')">
+                            <input type="checkbox" id="jira-count-tips" disabled>
+                            <span class="slider round"></span>
+                        </label>
+                    </p>
+                    <p></p>
+                </div>
+            </div>
```

Comment:
I'd suggest to move the `title` that explains the need for the optional permission up to the section heading "SAP Jira". It's more likely that people will notice it there than on the switch. Also, it holds for the whole SAP Jira section if there should be more options in the future

```suggestion
            <div class="section">
                <p class="section-heading" title="Optional: jira.tools.sap (You may enable this first within 'Optional Permissions')">SAP Jira</p>
                <div class="section-text disabled"
                    data-dynamic-content-script-name="jira.tools.sap"
                    title="Show the count tips on the right side of the pagination.">
                    <p>Show Count Tips</p>
                    <p><label class="switch">
                        <input type="checkbox" id="jira-count-tips" disabled>
                        <span class="slider round"></span>
                    </label></p>
                    <p></p>
                </div>
            </div>
```